lesson-2/practice/practice.md

```diff
@@ -0,0 +1,18 @@
+# Practice Exercises 7.2
```

Comment:
Generally, questions 2-7 are a bit too trivial, assuming that students have taken the time to engage with the material in the Learning Assignments section. I like question 8 a lot.

But now knowing which lessons performed well in Unit 6, when introducing a framework, I want to try and go with practice sets that help "bring it all together." AKA building up the simplest of CRUD apps over a 7-10 question practice set. This is fine though, since Lesson 3 does a bit more of that.

Reply:
That makes sense - can definitely look to include more of that into the Schema design lesson as well.